static/stable/prod/navigation/settings-navigation.json

```diff
@@ -88,6 +88,30 @@
           "appId": "learningResources",
           "title": "Learning Resources",
           "href": "/settings/learning-resources"
-      }
+      },
+      {
+          "id": "idmsvc",
+          "appId": "idmsvc",
+          "title": "Directory and Domain Services",
+          "description": "Register identity and access systems to enable machines to automatically join a domain",
+          "href": "/settings/idmsvc",
+          "icon": "PlaceholderIcon",
```

Comment:
I think we need also to restrict this to Preview mode, as we did in stage.  See PR https://github.com/RedHatInsights/chrome-service-backend/pull/674